content/about/partners.md

```diff
@@ -1,9 +1,25 @@
 ---
-title: Partner Clubs
+title: Partners
 layout: single
 # Card images should all be 348x196
 ---
 
+## Sponsors
+
+<div class="row">
+  <div class="col-lg-6 col-xl-4 mb-4 d-flex align-items-stretch">
+	{{< club title="Google" href="" src="/files/google_logo.png" id="google">}}
```

Comment:
Could you `href` to the Google uni recruiting page (or something else @wgervasio?)

Reply:
i think href to the recruiting page is good @kewbish